packages/auth-sveltekit/src/server.ts

```diff
@@ -0,0 +1,735 @@
+import {
+  redirect,
+  type Cookies,
+  type RequestEvent,
+  type ActionFailure,
+} from "@sveltejs/kit";
+import * as cookie from "cookie";
```

Comment:
I'm not sure this is needed? Sveltekit looks like it has it's own cookie handling functions, and the only place this seems to be used is in the `getSession` which I think could take a `RequestEvent` object instead, that has a `cookies` object.

Reply:
oh yes, I forgot to refactor this part, tnx